src/qt/button.cpp

```diff
@@ -46,7 +46,22 @@ bool wxButton::Create(wxWindow *parent, wxWindowID id,
 {     
     QtCreate(parent);
     SetLabel( wxIsStockID( id ) ? wxGetStockLabel( id ) : label );
-
+    
+    if( style & wxBU_LEFT )
+        m_qtPushButton->setStyleSheet( "Text-align:left" );
+    if( style & wxBU_RIGHT )
+        m_qtPushButton->setStyleSheet( "Text-align:right" );
+    if( style & wxBU_TOP )
+        m_qtPushButton->setStyleSheet( "Text-align:top" );
+    if( style & wxBU_BOTTOM )
+        m_qtPushButton->setStyleSheet( "Text-align:bottom" );
+    if( style & ( wxBU_LEFT | wxBU_RIGHT ) )
+        m_qtPushButton->setStyleSheet( "Text-align:center" );
+    if( style & ( wxBU_TOP | wxBU_BOTTOM ) )
+        m_qtPushButton->setStyleSheet( "Text-align:center" );
+    if( style & wxBORDER_NONE )
+        m_qtPushButton->setFlat( true );
```

Comment:
This is worth at least a separate commit (or if it already is one, github just won't show it to me). While it still is a style, it is otherwise unrelated to the previous changes.